cluster/addons/cluster-monitoring/google/heapster-controller.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 kind: ReplicationController
 metadata: 
   name: monitoring-heapster-v4
-  namespace: default
+  namespace: system
```

Comment:
I thought we agreed on "kubernetes" as the name?  I don't care much, but I don't want wires crossed.